storage/innobase/xtrabackup/src/backup_mysql.cc

```diff
@@ -2088,7 +2088,10 @@ char *make_argv(char *buf, size_t len, int argc, char **argv)
 		if (strncmp(*argv, "--password", strlen("--password")) == 0) {
 			arg = "--password=...";
 		}
-		if (strncmp(*argv, "--encrypt-key",
+                if (strncmp(*argv, "-p", strlen("-p")) == 0) {
+                  arg = "-p...";
+                }
+                if (strncmp(*argv, "--encrypt-key",
```

Comment:
plz check alignment. Also check new parameters that are passing cloud info credentials.

Reply:
thank you @satya-bodapati for cloud suggestion. would be adding xbcloud test-case as part of PXB-2723.